cubequery/tasks/__init__.py

```diff
@@ -298,6 +307,60 @@ def standard_validation(self, args):
 
         return errors
 
+
+def validate_standard_spatial_query(aoi, countries):
+    
+    errors = []
+
+    try:
+        parsed_polygon = wkt.loads(aoi)
+    except:
+        return [create_error_message({'id': 'aoi', 'error_message': 'Polygon could not be loaded',
+                                        '_comment': 'Polygon could not be loaded'})]
+
+    '''
+    Returns validity of geometery (bool)
+    * Whole of Fiji = True
+    * Suva = True
+    '''
+    valid_geom = parsed_polygon.is_valid

```

Comment:
Does this need to be stored? it might make more sense just having the is_valid in the if statement